api/openapi-spec/swagger.json

```diff
@@ -839,6 +843,10 @@
           "description": "Location in which all files related to the step will be stored (logs, artifacts, etc...). Can be overridden by individual items in Outputs. If omitted, will use the default artifact repository location configured in the controller, appended with the \u003cworkflowname\u003e/\u003cnodename\u003e in the key.",
           "$ref": "#/definitions/io.argoproj.workflow.v1alpha1.ArtifactLocation"
         },
+        "arguments": {
```

Comment:
Are you sure this belongs to the `Template` struct? AFAIK it should be in the `DagTask` struct where it already exists.